site/src/components/pages/user/block-list-item.tsx

```diff
@@ -18,10 +18,10 @@ export const BlockListItem = ({ block }: BlockListItemProps) => {
       title={block.displayName!}
       description={block.description}
       extraContent={
-        <Box display="flex" gap={1.5}>
-          <Link href={`/@${block.author}`}>@{block.author}</Link>
+        <Box display="flex" gap={1.5} component="span">
+          @{block.author}
```

Comment:
Fixes

```
Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.
```

Note that we no longer link to user inside a link to a block. I _guess_ it makes sense product-wise.

Reply:
Not sure if a link inside a link is a good UX and whether it is accessible. @nonparibus WDYT?

Reply:
For context, we’re talking about nested links in this list: https://blockprotocol.org/@hash/blocks

<img width="563" alt="Screenshot 2022-09-28 at 14 05 47" src="https://user-images.githubusercontent.com/608862/192786087-0147a9c4-4eb8-498b-ad82-67cfb62e8eb3.png">

Reply:
I removed author from `ListViewCard`. This component is only used at `/@user/blocks` or `/blocks` (i.e. _my blocks_).

https://blockprotocol-jcempba1y.stage.hash.ai/@hash/blocks

<img width="622" alt="Screenshot 2022-09-28 at 21 35 08" src="https://user-images.githubusercontent.com/608862/192883497-0bb5dfb2-e6a7-4e84-b142-ed69373bb92d.png">

---

I’m still not convinced that having `<a>` inside `<a>` produces good UX / a11y. If it was, Next.js would not warn about it:

```
Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.
```

It is possible to fool Next with `span` + `onClick`, but this won’t improve UI a11y. Happy to change this opinion if we can find examples in high-quality web apps.

Reply:
I defer on best implementation, but there are going to be times we want clickable islands within otherwise clickable elements that don't point to the same place.